### ISSUE TITLE
[Notes] Search Notes doesn't work

### ISSUE THREAD

**Issue by iamdannywyatt** [raycast/extensions#3652]
### Extension

https://www.raycast.com/tumtum/apple-notes

### Description

When I try to search for notes, I get this error:
![Screen Shot 2022-11-24 at 1 15 59 PM](https://user-images.githubusercontent.com/96640778/203843504-ec3183eb-36ff-46a5-a4f7-4fd0e5899245.png)

Here's the log:
[search notes log.txt](https://github.com/raycast/extensions/files/10086660/search.notes.log.txt)

**Comment by ron-myers** [comment]
Thank you for reporting this @iamdannywyatt 

Does this happen with all notes or just some? 

From the log it looks like you are selecting a note titled `C` and selecting it for open, does that sound correct?  The error in the log is:

```
osascript -e tell application "Notes" 
show note "C" 
end tell
23:41: execution error: Notes got an error: AppleEvent handler failed. (-10000)
```

Any chance you have multiple notes titled `C`?

**Comment by iamdannywyatt** [comment]
@ron-myers Thank you for the quick reply.
The issue is with any note, regardless of the name.
One thing though: now that you mentioned the "C note", that's not a note, it's a folder inside a folder called Passwords, but it's showing me as if it was a note, so that alone is already a bug.

I tried notes inside the default Notes folder (the one that can't be deleted), notes inside folders, notes inside folders that are inside folders. Everything shows me that error.

Even if I had multiple notes titles C (which is not the case anyway), it would show me both options and the moment I pick one, it should open that one, right? I assume each note has its own "id" or something like that, so I don't see that this could be the issue.
I tried with notes that have unique names and got the same error, so that's not the cause either.

**Comment by iamdannywyatt** [comment]
@ron-myers 
I tried it again after rebooting, just in case something was going wrong, but no luck.
One thing I noticed and I don't know if this is a normal behavior, but Notes was closed completely, and as soon as I typed "Search Notes" and hit Enter, it opened Notes without me even picking one option. So it showed me some notes on the list for like a second, but then it immediately opened Notes.
Also, for a few minutes when I would open the list of notes, it would show me a note that I deleted a long time ago, but it was showing me at the top of the list as if it was inside the Recently Deleted folder. So weird...

**Comment by ron-myers** [comment]
@iamdannywyatt  ok - I have found **an** issue - which may be your issue.  Do any of your folders have any non-alphanumeric characters?

The deleted items is a separate change that likely should be made.  I may add a feature to ignore deleted as part of correcting this issue.

**Comment by iamdannywyatt** [comment]
@ron-myers 
I have some folders that have hyphens such as **_To-do_** and I have one with an asterisk and the & symbol such as **_*Themes & Names_**.
Other than that, they are all pretty simple.
But as I mentioned, I tried finding notes inside the default "Notes" folder, which is just letters and I got the same result.

Are you saying that the fact that some folders have those characters/symbols affect the overall behavior, even if the notes I'm searching for are not inside those folders?
